.github/workflows/translate_lupdate.yml

```diff
@@ -37,9 +37,10 @@ jobs:
           bash ./buildscripts/ci/translation/run_lupdate.sh
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v6
+        uses: peter-evans/create-pull-request@v7
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
```

Comment:
As far as I can tell this is the default anyhow, isn't it`?

Reply:
Oh, you're right, I meant `author`...